ng2-components/ng2-activiti-form/src/components/activiti-form.component.spec.ts

```diff
@@ -634,4 +634,74 @@ describe('ActivitiForm', () => {
         expect(visibilityService.refreshVisibility).toHaveBeenCalledWith(field.form);
     });
 
+    it('should load form for ecm node', () => {
+        spyOn(formComponent, 'loadFormForEcmNode').and.stub();
+
+        formComponent.nodeId = '<id>';
+        formComponent.ngOnInit();
+
+        expect(formComponent.loadFormForEcmNode).toHaveBeenCalled();
```

Comment:
Perhaps this needs further discussion among the team, then. My position is to have a minimal public interface on the class to allow for later refactoring without having to re-write the tests and ensures we can commit to not break public/protected methods that others bind to in a later (non-major) release. There should be ways to avoid this increasing complexity of the tests too much, e.g. refactoring private methods into external classes which can then be tested themselves via their public methods.